variables.js

```diff
@@ -2,36 +2,50 @@ const variables = {
 	cloud: {
 		prodname: "Calico Cloud",
 		prodnamedash: "calico-cloud",
+		docsPathPrefix: [
```

Comment:
Do we really want to store it in variables? Looks like it's used only for plugin purposes.

Reply:
I think it's better to keep these values in variables.js since the paths are tightly associated with those variables. This is what allows us support the overriding dynamically.